jobs/build/ocp4/Jenkinsfile

```diff
@@ -119,9 +121,30 @@ node {
                 stage("initialize") { build.initialize() }
                 buildlib.assertBuildPermitted(doozerOpts)
                 stage("build RPMs") { build.stageBuildRpms() }
-                lock("compose-lock-${params.BUILD_VERSION}") {
-                    stage("build compose") { build.stageBuildCompose() }
+
+                // if the automation is not frozen perform compose
+                // otherwise if automation is frozen but there
+                // are rpms in the build plan perform compose
+                // and announce on slack
+
+                if(buildlib.isBuildPermitted(doozerOpts)){
```

Comment:
this is true if a human is running it and `scheduled` is frozen. we need a further refinement on `isBuildPermitted` to let us find out if we're running a non-automated build during an automation freeze.